frontend/packages/dev-console/src/components/PodsRing.tsx

```diff
@@ -0,0 +1,94 @@
+import * as React from 'react';
+import { VictoryLabel } from 'victory';
+import * as _ from 'lodash';
+import { K8sResourceKind, k8sPatch, K8sKind } from '@console/internal/module/k8s';
+import { AngleUpIcon, AngleDownIcon } from '@patternfly/react-icons';
+import PodStatus from './topology/shapes/PodStatus';
+import { Pod } from './topology/topology-types';
+import SvgDefsProvider from './svg/SvgDefsProvider';
+
+interface PodsRingProps {
+  pods: Pod[];
+  obj: K8sResourceKind;
+  resourceKind: K8sKind;
+  path: string;
+  ringContainerSize: number;
+}
+
+const PodsRing: React.FC<PodsRingProps> = ({
+  pods,
+  obj,
+  resourceKind,
+  path,
+  ringContainerSize,
+}) => {
+  const [podCount, setPodCount] = React.useState(0);
+  const [scaleTo, setScaleTo] = React.useState(0);
+  const [isScaling, setIsScaling] = React.useState(false);
+
+  const size = ringContainerSize - 80;
+  const radius = size / 2;
+  const podStatusStrokeWidth = (8 / 104) * size;
+  const podStatusInset = (5 / 104) * size;
+  const podStatusOuterRadius = radius - podStatusInset;
+  const podStatusInnerRadius = podStatusOuterRadius - podStatusStrokeWidth;
+
+  React.useEffect(() => {
+    setPodCount(obj.spec.replicas);
+  }, [podCount, obj, pods]);
+
+  const handleScaling = (operation: number) => {
+    setIsScaling(true);
+    setScaleTo(podCount + operation);
+    const patch = [{ op: 'replace', path, value: _.toInteger(podCount + operation) }];
+    const promise: Promise<K8sResourceKind> = k8sPatch(resourceKind, obj, patch);
+    promise
+      .then(() => setIsScaling(false))
+      .catch((error) => {
+        throw error;
+      });
+  };
+
+  return (
+    <svg width={ringContainerSize} height={ringContainerSize}>
+      <SvgDefsProvider>
+        <g transform="translate(0, 35)">
+          <PodStatus
+            innerRadius={podStatusInnerRadius}
+            outerRadius={podStatusOuterRadius}
+            data={pods}
+            size={size}
+          />
+          <VictoryLabel
```

Comment:
The fonts don't match the rest of the page. Should be overpass.
![image](https://user-images.githubusercontent.com/14068621/61540485-b3d5b480-aa0b-11e9-939e-04a9ac9ac000.png)